doc/dev/index.rst

```diff
@@ -179,6 +179,100 @@ with::
 
 For more ``vstart.sh`` examples, see :doc:`/dev/quick_guide`.
 
+What is merged where and when ?
+===============================
+
+Commits are merged into branches according to criterions that change
+during the lifecycle of a Ceph release. This chapter is the inventory
+of what can be merged in which branch at a given point in time.
+
+Development releases (i.e. x.0.z)
+---------------------------------
+
+What ?
+^^^^^^
+
+ * features and bug fixes
+
+Where ?
+^^^^^^^
+
+Features are merged to the master branch. Bug fixes should be (but it is not mandatory)
+merged in the branch of the stable release (hammer for 0.94.x, infernalis for
+9.2.x, etc.).
+
+When ?
+^^^^^^
+
+After the stable release candidates of the previous release starts phase 2 (see below).
+
+
+Stable release candidates (i.e. x.1.z) phase 1
+----------------------------------------------
+
+What ?
+^^^^^^
+
+ * bug fixes only
+
+Where ?
+^^^^^^^
+
+The branch of the stable release (hammer for 0.94.x, infernalis for
+9.2.x, etc.) or master.  All commits in master will be merged in the
+branch of stable release. All commits in the branch of stable release
+will be merged in master. In other words, there is no difference
+between master and the stable release branch.
+
+When ?
+^^^^^^
+
+After the first stable release candidate is published, i.e. after the
+x.1.0 tag is set in the release branch.
+
+
+Stable release candidates (i.e. x.1.z) phase 2
+----------------------------------------------
+
+What ?
+^^^^^^
+
+ * bug fixes only
+
+Where ?
+^^^^^^^
+
+The branch of the stable release (hammer for 0.94.x, infernalis for
+9.2.x, etc.) or master.
```

Comment:
For example, Sage Weil announces the beginning of jewel release candidates phase 2. From this point on, bug fixes are merged to "kraken" (or master) and may be flagged for backport to jewel. They will not make it into the first jewel release (`10.2.0`).